### ISSUE TITLE
This import is never used as a value and must use 'import type' because 'importsNotUsedAsValues' is set to 'error'.

### ISSUE THREAD

**Issue by codatorz** [themesberg/flowbite-vue#165]
when using
```
<script lang="ts" setup>
import { Tabs, Tab } from "flowbite-vue";
...
```

```
this are my dependencies: devDependencies": {
    "@rushstack/eslint-patch": "^1.1.4",
    "@tailwindcss/forms": "^0.5.3",
    "@types/jsdom": "^20.0.1",
    "@types/node": "^18.11.12",
    "@types/qs": "^6.9.7",
    "@vitejs/plugin-vue": "^4.0.0",
    "@vue/eslint-config-prettier": "^7.0.0",
    "@vue/eslint-config-typescript": "^11.0.0",
    "@vue/test-utils": "^2.2.6",
    "@vue/tsconfig": "^0.1.3",
    "autoprefixer": "^10.4.13",
    "cypress": "^12.0.2",
    "eslint": "^8.22.0",
    "eslint-plugin-cypress": "^2.12.1",
    "eslint-plugin-vue": "^9.3.0",
    "jsdom": "^20.0.3",
    "npm-run-all": "^4.1.5",
    "postcss": "^8.4.21",
    "prettier": "^2.7.1",
    "start-server-and-test": "^1.15.2",
    "tailwindcss": "^3.2.4",
    "typescript": "~4.7.4",
    "vite": "^4.0.0",
    "vitest": "^0.25.6",
    "vue-tsc": "^1.0.12"
  }
```

if I use `<script setup> ` I don't get the error but I need to use typescript, 
if I use `import  type { Tabs, Tab } from "flowbite-vue";` then the component doesn't work 
do I need to change `importsNotUsedAsValues`?
or what can i do about it?

by the way I have no issue with just flowbite, I mean `import { Modal } from "flowbite";` doesn't show an error.

thank you in advance.

**Comment by codatorz** [comment]
ok, using:
```
/*@ts-ignore*/
import { Tabs, Tab } from "flowbite-vue";
```
was a patch but also I was using <tabs></tabs> instead of <Tabs></Tabs> so I changed it and the error was gone but if someone could still help with the explanation for that I'll appreciate it,

thank you!.

**Comment by cogor** [comment]
Hey, I fixed this error, wait for next release(0.0.16)

**Comment by codatorz** [comment]
thank you!
